src/modules/emote_menu/utils/twitch-emotes.js

```diff
@@ -113,6 +170,20 @@ export async function loadTwitchEmotes() {
         newToken = emoteToken;
       }
 
+      let predicate = () => false;
+      if (['SUBSCRIPTIONS', 'FOLLOWER'].includes(type)) {
```

Comment:
is this logic future proof? should we be checking for any emote which isn't in the available emote sets?